zappa/cli.py

```diff
@@ -422,7 +422,7 @@ def schedule(self):
                 events = []
             keep_warm_rate = self.zappa_settings[self.api_stage].get('keep_warm_expression', "rate(5 minutes)")
             events.append({'name': 'zappa-keep-warm',
-                           'function': self.lambda_handler,
+                           'function': 'handler.heater',
```

Comment:
Ahhh, hm, okay, I see your point. I don't think it needs its own empty function though. The point is that it should actually call the default WSGI handler so that any imports etc that need to happen inside the application are warm as well. I don't think imports are kept in cache unless they are explicitly made.

"`heater`" is cute but I think ultimately confusing: keep_warm_callback maybe? Is there actually a use case for this though, versus just scheduling a custom function? I don't want to add advanced_settings unless they're actually useful.